libs/designsystem/item/src/item.component.stories.ts

```diff
@@ -22,6 +23,9 @@ const meta: Meta<ItemComponent> = {
     }),
   ],
   title: 'Components / Item',
+  parameters: {
+    chromatic: { modes: { ...responsiveModes } },
+  },
```

Comment:
We should be able to add this on the story level, for the specific story, instead of all the stories of this file.